scripts/run_unit_tests.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+CUR_DIR=$(dirname "$0")
+TEST_LIST=$("$CUR_DIR"/for-each-module.sh "go list ./..." | grep -v -e "sample" | grep -e "^github")
+# shellcheck disable=SC2086
+gotestsum --junitfile ~/junit/unit-tests.xml -- -short $TEST_LIST
```

Comment:
I think we need a file per folder, or it unify all results?

Reply:
It is unify all results

Reply:
@denis-tingajkin is right, it is unify all results

Reply:
ok